favorites.py

```diff
@@ -179,10 +198,11 @@ def load_favorite_files(cls, obj, force=False, clean=False, win_id=None):
         # Is project enabled
         FavProjects.project_adjust(obj, win_id, force)
 
+        cls.check_aliases(obj.file_name)
         if not exists(obj.file_name):
             if force:
                 # Create file list if it doesn't exist
-                if cls.create_favorite_list(obj, {"version": 1, "files": [], "groups": {}}, force=True):
+                if cls.create_favorite_list(obj, {"version": 1.1, "files": [], "groups": {}}, force=True):

```

Comment:
Lets just call it version `2`.  It is unlikely for formats that I am dong a `0.0.0` scheme.  I may use `0.0`, but this is clearly a major change, so let's call it `2`.